testkit/src/main/scala/org/allenai/common/testkit/SingeInstanceUnitSpec.scala

```diff
@@ -0,0 +1,6 @@
+package org.allenai.common.testkit
+
+/** Base class for any unit test that, for some reason, want to share the Spec instance.
+  * Usually the reason is a slow expensive test setup time (setting up a 3rd party DB, etc.)
+  */
+abstract class SingeInstanceUnitSpec extends AllenAiBaseSpec
```

Comment:
Would be good to have a `IntegrationSpec` too. I'd rather we kept it separate from the `SingleInstanceUnitSpec`. Could just be :

```scala
abstract class IntegrationSpec extends AllenAiBaseSpec
```